recipe/conda_build_config.yaml

```
@@ -2,9 +2,27 @@ c_compiler_version:        # [linux]
   - 12                     # [linux]
 cxx_compiler_version:      # [linux]
   - 12                     # [linux]
-arm_variant_type:          # [aarch64]
-  - sbsa                   # [aarch64]
+arm_variant_target:
+  - None
+  - tegra                  # [linux]
+  - sbsa                   # [linux]
 cross_target_platform:
+  - None           # [not (linux or win)]
   - linux-64       # [linux]
   - linux-aarch64  # [linux]
+  - linux-aarch64  # [linux]
   - win-64         # [win]
+DEFAULT_CUDAARCHS:
+  - "50-real;52-real;60-real;61-real;70-real;75-real;80-real;86-real;89-real;90"
+  - "72-real;87"  # [linux]
+  - "50-real;52-real;60-real;61-real;70-real;75-real;80-real;86-real;89-real;90"  # [linux]
+DEFAULT_TORCH_CUDA_ARCH_LIST:
+  - "5.0;5.2;6.0;6.1;7.0;7.5;8.0;8.6;8.9;9.0+PTX"
+  - "7.2;8.7+PTX"  # [linux]
+  - "5.0;5.2;6.0;6.1;7.0;7.5;8.0;8.6;8.9;9.0+PTX"  # [linux]
```

There are a couple architectures we are missing for Tegra. Have added them below

Also since we are now on CUDA 12.8, which has Blackwell support, think we should be including that in the architectures. This is also important as we migrate for CUDA 12.8

```suggestion
DEFAULT_CUDAARCHS:
  - "50-real;52-real;60-real;61-real;70-real;75-real;80-real;86-real;89-real;90-real;100-real;120"
  - "50-real;52-real;60-real;61-real;70-real;75-real;80-real;86-real;89-real;90-real;100-real;120"  # [linux]
  - "53-real;62-real;72-real;87"  # [linux]
DEFAULT_TORCH_CUDA_ARCH_LIST:
  - "5.0;5.2;6.0;6.1;7.0;7.5;8.0;8.6;8.9;9.0;10.0;12.0+PTX"
  - "5.0;5.2;6.0;6.1;7.0;7.5;8.0;8.6;8.9;9.0;10.0;12.0+PTX"  # [linux]
  - "5.3;6.2;7.2;8.7+PTX"  # [linux]
```

---

The newer archs (>=10) are missing because they weren't officially announced when I started working on this PR.

The tegra archs I specifically chose based on a synthesis of the tables linked in https://github.com/conda-forge/cuda-feedstock/issues/57#issuecomment-2576164789. My understanding is that tegra devices can only get the CTK/driver through Jetpack which means devices are limited based on their support status with Jetpack. It follows that our channel does not support tegra devices with compute capability <7 because we don't support CTK 10 anymore. Summary tables to follow.